WorkloadManagementSystem/Service/PilotAgentsHandler.py

```diff
@@ -0,0 +1,60 @@
+""" PilotAgentsHandler is the implementation of the PilotAgents service
+
+    The following methods are available in the Service interface
+
+    addPilotsLogging()
+    getPilotsLogging
+    setPilotsUUIDtoIDMapping()
+    addPilotsUUID()
+    deletePilotsLogging()
+
+"""
+
+__RCSID__ = "$Id: $"
+
+from DIRAC.WorkloadManagementSystem.DB.PilotAgentsDB import PilotAgentsDB
+
+import types
+from DIRAC.Core.DISET.RequestHandler import RequestHandler
+from DIRAC import gLogger, S_OK, S_ERROR
+from DIRAC.Core.Utilities import Time
+# from DIRAC.WorkloadManagementSystem.DB import PilotAgentsDB
+
+class PilotAgentsHandler( RequestHandler ):
+  
+  @classmethod
+  def initializeHandler( cls, serviceInfo ):
+    return S_OK()
+  
+  def initialize(self):
+    self.pilotAgents = PilotAgentsDB()
+    
+  auth_addPilotsLogging = [ 'all' ]
+  types_addPilotsLogging = [ types.StringType, types.StringType, types.StringType, types.FloatType, types.StringType ]
+  def export_addPilotsLogging( self, pilotUUID, status, minorStatus, timeStamp, source ):
+    
+    return self.pilotAgents.addPilotsLogging( pilotUUID, status, minorStatus, timeStamp, source )
+  
+  auth_getPilotsLogging = [ 'all' ]
```

Comment:
The auth rules should be revised here. 

I would say that "authenticated" is the minimal authorization level, but for "set" and especially "delete" methods we should have "Operator" as base.